pleskdistup/actions/extensions.py

```diff
@@ -146,21 +146,27 @@ def __init__(self) -> None:
         self.ext_name = "tuxcare-els"
 
     def _is_required(self) -> bool:
-        # If database is not ready, we can't be sure about installed extensions
-        if (not plesk.is_plesk_database_ready()):
+        try:
+            return self.ext_name in dict(plesk.list_installed_extensions())
+        except plesk.PleskDatabaseIsDown:
+            # If database is not ready we will not be able to uninstall the extension anyway
             return False
 
-        return self.ext_name in dict(plesk.list_installed_extensions())
-
     def _prepare_action(self) -> action.ActionResult:
-        plesk.uninstall_extension(self.ext_name)
+        try:
+            plesk.uninstall_extension(self.ext_name)
+        except plesk.PleskDatabaseIsDown:
+            log.warn("Removing TuxCare ELS extension called when database is already down")
```

Comment:
This is written like it's intended for a developer. For a user it's unclear what to do with this.

Reply:
This message is intended for developers. It indicates that the order of actions is probably incorrect or that previous actions have failed. It's unlikely that a user will be able to fix the problem on their own.